test/e2e/jest.config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"rootDir": "../../",
+	"preset": "@wordpress/jest-preset-default",
+	"setupFiles": [],
```

Comment:
I think `support/bootstrap` should go in here or we should use the existing `jest-puppeteer` preset: https://www.npmjs.com/package/jest-puppeteer. See also: https://facebook.github.io/jest/docs/en/puppeteer.html#use-puppeteer-preset which informs about the outdated preset.

Reply:
I tried the bootstrap here but it didn't work, I can't remember the reason. I think it's related to `beforeAll` not yet defined or something like that. Any idea?

Reply:
In other examples, it goes to the `globalSetup`, but they also do some optimizations to share a connection to the browser between tests. We should look into it later.

Reply:
We probably shouldn't use this preset as `setupFiles` only appends new files, it doesn't allow to override it. Sometimes it's good but sometimes bad ...

Reply:
I'm keen on using the `jest-puppeteer` preset but it is pretty new and shiny.

Currently an issue https://github.com/smooth-code/jest-puppeteer/issues/17 _"Jest's native expect functions are broken by puppeteer-expect"_ would be a **blocker** for us

Reply:
@ntwb, another blocker is that whatever they suggest in the example repository recommended by Jest docs, it doesn't work stable with the tests we have :(
It looks like what is proposed in this PR is the way to go.